eng/pipelines/templates/jobs/ci.tests.yml

```diff
@@ -24,6 +24,9 @@ parameters:
 - name: CloudConfig
   type: object
   default: {}
+- name: AdditionalTestArguments
```

Comment:
Where is this parameter actually being used? The only usage I see below is variable access and not template parameter access.

Reply:
I suspect you just want to set the default value to empty string in globals.yml like we did with the old parameter.